linalg/ode.cpp

```diff
@@ -11,6 +11,9 @@
 
 #include "operator.hpp"
 #include "ode.hpp"
+#ifdef MFEM_USE_MPI
+#include "../general/communication.hpp"
+#endif
```

Comment:
I don't think the `#ifdef` guard is needed.

Reply:
I was not sure. I was a bit struggling with the making sure only root was outputting. I will remove the guard and check if things still work.

Reply:
Done